tools/internal/pkg/smb/smb.go

```diff
@@ -68,7 +68,7 @@ func getSubBenchmarks(cfg *benchmark.Config, wp *workspace.Config) map[string]ap
 		URL:     "file:///" + filepath.Join(smbDir, "src", mpiOverheadID),
 		BinName: mpiOverheadBinName,
 		BinPath: filepath.Join(wp.InstallDir, mpiOverheadID, mpiOverheadID, mpiOverheadBinName),
-		BinArgs: []string{"--msgsize", fmt.Sprintf("%d", defaultMsgSize)},
+		BinArgs: []string{" --msgsize", " ", fmt.Sprintf("%d", defaultMsgSize)},
```

Comment:
go_hpc_jobmgr v1.3.3 automatically add a space between the binary and the args and a space between args so i do not think this is required